Add Sun Valley 10 Action Center theme

milestprower92 opened the pull request:
This theme tries to recreate the design of the hidden 10 Action Center UI on Windows 10 builds 18945-18947, along by adding the newer WinUI design to the old Windows 10 AC which includes:
* Rounded corners
* Windows 11-like textboxes, scrollbars, buttons, tooltips, comboboxes, sliders, etc
* Quick action gripper
* Dynamic sized notification panel regarding the amount of notifications

## Windows 11 (leftover Windows 10 Action Center
<img width="408" height="899" alt="screenshot" src="https://github.com/user-attachments/assets/8c4009a5-a844-4759-82d3-cc88f73b49f4" />

## Windows 10
<img width="427" height="329" alt="screenshot2" src="https://github.com/user-attachments/assets/a69a9faf-7af1-4c87-b7a2-8bbceba100ce" />

milestprower92 committed:
Create SunValley

milestprower92 committed:
Delete Themes/SunValley

milestprower92 committed:
Create README.md

milestprower92 committed:
Add README for Sun Valley Action Center theme

This README provides an overview of the Sun Valley Action Center theme, detailing its features, installation instructions, and manual import steps.

milestprower92 committed:
Add files via upload

milestprower92 committed:
Add Windows 10 Action Center themes section

Added a section for Windows 10 Action Center themes.

milestprower92 committed:
Update link text for Windows 10 Action Center themes

milestprower92 committed:
Add files via upload

milestprower92 committed:
Delete Themes/SunValley/screenshot-small.png

too small

milestprower92 committed:
Add files via upload

milestprower92 committed:
Update README with Windows 10 and 11 screenshots

Added screenshots for Windows 10 and Windows 11 sections.

milestprower92 committed:
Add files via upload

SandTechStuff commented on Themes/SunValley/README.md:
```diff
@@ -0,0 +1,334 @@
+"controlStyles[11].styles[18]":"Background@Pressed:=<SolidColorBrush Color=\"{ThemeResource SystemChromeAltLowColor}\" Opacity=\"0.1\" />",
+"controlStyles[11].styles[19]":"Padding=6",
+"controlStyles[11].styles[20]":"Foreground@Checked=Black",
+"controlStyles[11].styles[21]":"Foreground@CheckedPointerOver=Black","controlStyles[11].styles[22]":"Foreground@CheckedPressed=Black",
```
```suggestion
"controlStyles[11].styles[21]":"Foreground@CheckedPointerOver=Black",
"controlStyles[11].styles[22]":"Foreground@CheckedPressed=Black",
```

Missing a linebreak here

SandTechStuff commented on Themes/SunValley/README.md:
```diff
@@ -0,0 +1,334 @@
+"controlStyles[56].styles[1]":"BorderBrush:=<AcrylicBrush TintColor=\"{ThemeResource SystemChromeHighColor}\" TintOpacity=\"0.1\" FallbackColor=\"{ThemeResource SystemChromeMediumHighColor}\" />",
+"controlStyles[56].styles[2]":"BorderThickness=1",
+"controlStyles[56].styles[3]":"CornerRadius=7",
+"controlStyles[57].target":"ActionCenter.NotificationListView#MainListView > Border > ScrollViewer > Border > Grid > Windows.UI.Xaml.Controls.Primitives.ScrollBar > Grid > Grid > Windows.UI.Xaml.Controls.Primitives.Thumb > Rectangle#ThumbVisual","controlStyles[58].target":"ActionCenter.NotificationListView#MainListView > Border > ScrollViewer > Border > Grid > Windows.UI.Xaml.Controls.Primitives. ...
```
```suggestion
"controlStyles[57].target":"ActionCenter.NotificationListView#MainListView > Border > ScrollViewer > Border > Grid > Windows.UI.Xaml.Controls.Primitives.ScrollBar > Grid > Grid > Windows.UI.Xaml.Controls.Primitives.Thumb > Rectangle#ThumbVisual",
"controlStyles[58].target":"ActionCenter.NotificationListView#MainListView > Border > ScrollViewer > Border > Grid > Windows.UI.Xaml.Controls.Primitives.ScrollBar > Grid > Grid > Windows.UI.Xaml.Controls.Primitives.RepeatButton#VerticalSmallDecrease > Grid > FontIcon#Arrow",
```

Missing a linebreak here

milestprower92 committed:
Update Themes/SunValley/README.md

Co-authored-by: Sand <164842290+SandTechStuff@users.noreply.github.com>

TheGamer1445891 commented:
there's a typo

milestprower92 committed:
Merge branch 'ramensoftware:main' into main

milestprower92 committed:
Rename theme from 10JumpLists to 10Flyouts

milestprower92 committed:
Create README.md

milestprower92 committed:
Update README.md with theme usage instructions

Added detailed instructions for theme selection and manual installation.

milestprower92 committed:
Add files via upload

milestprower92 committed:
Fix screenshot link for 10Flyouts theme

milestprower92 committed:
Update screenshot link for 10Flyouts theme

milestprower92 committed:
Add files via upload

milestprower92 committed:
Delete Themes/10JumpLists directory

milestprower92 committed:
Add control styles for 10Flyouts theme

Added various control styles for the 10Flyouts theme, including adjustments for buttons, text blocks, and other UI elements.
